src/renderer/mod.rs

```diff
@@ -307,9 +313,22 @@ impl Renderer {
         };
 
         let settings = self.settings.get::<RendererSettings>();
+        let size = root_canvas.base_layer_size();
+        let screen_rect = Rect::from_xywh(0.0, 0.0, size.width as f32, size.height as f32);
         let root_window_regions = root_windows
             .into_iter()
-            .map(|window| window.draw(root_canvas, default_background, grid_scale))
+            .map(|window| {
+                window.draw(
+                    root_canvas,
+                    (opacity * 255.0) as u8,
+                    Background {
+                        color: default_background.with_a((255.0 * background_transparency) as u8),
+                        image: background_image,
+                    },
+                    &screen_rect,
+                    grid_scale,
+                )
+            })
```

Comment:
This is probably also a problem with this code, the normal opacity setting does not work properly (the right is Neovide main)
![image](https://github.com/user-attachments/assets/d148c601-03b8-4740-ad88-b8d84089f1b6)

Without a background image it does not function at all
![image](https://github.com/user-attachments/assets/ac4268da-65e9-4e6b-a632-13e38d5c748a)